src/content/wrapped-ether.md

```diff
@@ -0,0 +1,121 @@
+
+Transferring ETH to another account requires creating a `send` transaction with the recipient's address as the destination (after signing with your [private key](/glossary/#private-key) to prove ownership of funds). As with any other Ethereum transaction, sending ETH demands paying a gas fee which is demoninated in [gwei](/glossary/#gwei) (a unit of Ether).
+
+Unlike ETH, the ownership and transfer of WETH tokens is handled by the [WETH token contract](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2). Balances of WETH token owners are stored **inside the contract** as key-value pairs where a key (an address) maps to a value (the total tokens owned by an account). As such, a wallet application cannot automatically track changes in your WETH balance unless you explictly ask the wallet application to "watch" the WETH token contract (i.e., read the contract's state).  
+
+Sending WETH tokens also requires a different approach: you'll have to create a transaction with the WETH token contract's address (not the recipient's address) as the destination. This transaction calls the smart contract's `transfer` function with the recipient's address and amount of tokens to transfer supplied as transaction parameters. While this transaction is recorded on the blockchain, the actual transfer of WETH tokens from one address to another occurs inside the token contract and isn't recorded directly on the blockchain (hence, it is described as an [internal transaction](https://coinmarketcap.com/alexandria/glossary/internal-transaction)). 
+
+We should also mention that you cannot pay gas for a transaction transferring WETH using WETH tokens. This may sound confusing, but less so once you understand that the Ethereum protocol specifies Ether (ETH) as the only currency for paying for computation on the Ethereum network. This is why you are advised to have some ETH in your wallet when wrapping/unwrapping ETH or sending WETH to another address. 
+
+### Uses of ETH vs WETH {#eth-vs-weth-use-cases} 
+
+Ether is primarily used for transferring value between users and compensating [nodes](/glossary/#node providing computational resources on the network. ETH is also used to secure the Ethereum network: validators processing transactions and producing new blocks must [deposit a bond paid in ETH](/staking/) before assuming this role. If a validator acts maliciously and attempts to subvert protocol rules, their ETH deposit will be [slashed](/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/#slashing) (destroyed) as punishment. 
```

Comment:
"Nodes" are not rewarded - only validators.
```suggestion
Ether is primarily used for transferring value between users and compensating validators securing the network. ETH is also used as collateral by validators. They stake ETH into a smart contract so that it can be destroyed if they act maliciously. 
```